lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_utilities.py

```diff
@@ -161,7 +161,7 @@ def create_cube_with_percentiles(percentiles, template_cube, cube_data,
             template cube.
 
     """
-    percentile_coord_name = custom_name or 'percentile_over_realization'
+    percentile_coord_name = custom_name or 'percentile'
```

Comment:
I'm sorry, I've just noticed "custom_name" isn't in the docstring. I know it isn't your change but would you mind fixing it? Also cube_units is also missing from the docstring

Reply:
Thanks for fixing this